common/network-common/src/main/java/org/apache/spark/network/util/LevelDBProvider.java

```diff
@@ -85,14 +84,6 @@ public static DB initLevelDB(File dbFile, StoreVersion version, ObjectMapper map
     return tmpDb;
   }
 
-  @VisibleForTesting
-  static DB initLevelDB(File file) throws IOException {
-    Options options = new Options();
-    options.createIfMissing(true);
-    JniDBFactory factory = new JniDBFactory();
-    return factory.open(file, options);
-  }
-
```

Comment:
This is mainly used for tests, any particular reason to remove it ?
(There is a lack of error handling in this method, and would fail in case of corruption or other issues - which I believe `YarnShuffleIntegrationSuite` is leveraging)

Reply:
Yes, this method is only called by `ShuffleTestAccessor#reloadRegisteredExecutors`. Currently, it is only used for one test case to verify a special test scenario:

https://github.com/apache/spark/blob/6c3c9575cf06fb0537b22909d784bc6bc1d61b85/resource-managers/yarn/src/test/scala/org/apache/spark/deploy/yarn/YarnShuffleIntegrationSuite.scala#L152-L170

This method  was originally in `ShuffleTestAccessor` and it was moved to `LevelDBProvider` in the previous PR, but it is not a general method, so I want move it back to `ShuffleTestAccessor`